scripts/common/common.sh

```diff
@@ -1178,17 +1178,16 @@ function check_for_running_pods() {
     for pod in $ns_pods; do
         status=$(kubectl get pod "$pod" -n "$namespace" -o jsonpath='{.status.phase}')
 
-        # determine if pod is manged by a Job
-        if kubectl get pod "$pod" -n "$namespace" -o jsonpath='{.metadata.ownerReferences[*].kind}' | grep -q "Job"; then
-            is_job_controller=1
-        fi
-
         # ignore pods that have been Evicted
         if [ "$status" == "Failed" ] && [[ $(kubectl get pod "$pod" -n "$namespace" -o jsonpath='{.status.reason}') == "Evicted" ]]; then
             continue
         fi
 
-        if [ "$status" != "Running" ] && [ "$status" != "Succeeded" ] && [ "$status" != "Completed" ]; then
+        if [ "$status" == "Succeeded" ]; then
+            continue
+        fi
+
+        if [ "$status" != "Running" ]; then
```

Comment:
What about the Completed status ? Isn't it necessary anymore ?

Reply:
It wasn't a thing - the status for completed pods is "Succeeded"